bower.json

```diff
@@ -28,13 +28,27 @@
     "maps"
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.7.1",
-    "google-apis": "GoogleWebComponents/google-apis#^1.1.7",
-    "iron-resizable-behavior": "PolymerElements/iron-resizable-behavior#^1.0.5",
-    "iron-selector": "PolymerElements/iron-selector#^1.5.2"
+    "polymer": "Polymer/polymer#1.9 - 2",
+    "google-apis": "GoogleWebComponents/google-apis#2.0-preview",
```

Comment:
@e111077 Hm, this is very strange. Seems you're wrong. In line 44 we use v1 of `google-apis`. All this time before yesterday this pull request was merged, `google-map` used v1 of `google-apis` and it worked like a charm.